src/main/kotlin/org/onflow/flow/sdk/impl/FlowAccessApiImpl.kt

```diff
@@ -81,6 +81,45 @@ class FlowAccessApiImpl(
         }
     }
 
+    override fun getTransactionsByBlockId(id: FlowId): List<FlowTransaction>? {
+        val ret = api.getTransactionsByBlockID(
+            Access.GetTransactionsByBlockIDRequest.newBuilder()
+                .setBlockId(id.byteStringValue)
+                .build()
+        )
+        return if (ret.transactionsList.isNotEmpty()) {
+            ret.transactionsList.map { FlowTransaction.of(it) }
+        } else {
+            null
+        }
+    }
```

Comment:
I agree with the bot, null returns are somewhat of an anti-pattern

Reply:
@franklywatson Yes I noticed this as well, will go ahead and introduce this in a follow up PR as all the Access API methods in the SDK would be affected and it can be a breaking change. Thanks!